spec/features/idv/doc_auth/ssn_step_spec.rb

```diff
@@ -17,15 +17,17 @@
       expect(page).to have_content(t('doc_auth.headings.capture_complete'))
     end
 
-    it 'proceeds to the next page with valid info' do
+    it 'proceeds to the next page with valid info', js: true do
       fill_out_ssn_form_ok
+      expect(page.find('#doc_auth_ssn')['aria-invalid']).to eq('false')
       click_idv_continue
```

Comment:
Since this is now running in JavaScript, there could be a brief delay before the redirect and I might worry the assertion could fail.

Maybe use our new `wait` argument?

```suggestion
      click_idv_continue(wait: true)
```

Reply:
Seems worthwhile! Will update